extensions/src/main/java/com/redhat/lightblue/extensions/sequance/SequanceSupport.java

```diff
@@ -0,0 +1,15 @@
+package com.redhat.lightblue.extensions.sequance;
+
+import com.redhat.lightblue.extensions.Extension;
+
+/**
+ * Sequence extension provides backend specific sequence number generation capabilities.
+ *
+ * @author mpatercz
+ *
+ */
+public interface SequanceSupport extends Extension {
+
+    Sequance getSequanceInstance();
+
```

Comment:
Should this be getSequenceInstance(String name) where name is datasource, or backend?